WTWSMS/history/characters/hunnic.txt

```diff
@@ -172,8 +171,8 @@
 	trait="envious"
 	trait="brilliant_strategist"
 	father = 73
-	350.1.2 = {
-		birth="350.1.2"
+	368.1.2 = {
+		birth=yes
```

Comment:
No, this doesn't work, Octar was younger than Mundzuk, and absolutely not his twin.

Reply:
? In master branch he is much older than Mudzuk.
I will edit his birthdate further.

Reply:
Done, made him 10 years younger.